package.json

```diff
@@ -17,9 +17,7 @@
   },
   "homepage": "https://github.com/Clever/discovery-node",
   "devDependencies": {
-    "mocha": "^2.3.2"
-  },
-  "dependencies": {
+    "mocha": "^2.3.2",
     "coffee-script": "^1.10.0"
```

Comment:
I like how you all caught this ;p

Yeah, it's still needed for tests. The auto-generated code was ugly and it would have taken longer to compile it by hand.